### PR TITLE
bump pdftranslator to mupdf 1.19 compatible version

### PR DESCRIPTION
**Pull request by ivanholmes**
related to #6823

### CODE REVIEW

File: haiku-apps/pdftranslator/pdftranslator-1.0.4.recipe [hunk 1]
```diff
@@ -13,24 +13,24 @@ COPYRIGHT="2012-2022 3dEyes**"
...
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	pdftranslator$secondaryArchSuffix = $portVersion
 	addon:PDFTranslator$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libmupdf$secondaryArchSuffix
+	lib:libmupdf$secondaryArchSuffix >= 1.19
```
Comment (Begasus) [inline haiku-apps/pdftranslator/pdftranslator-1.0.4.recipe:28]: Only needed for the developer package, PS, thanks for upstream patch! :+1: 
```diff
- 	lib:libmupdf$secondaryArchSuffix >= 1.19
+ 	lib:libmupdf$secondaryArchSuffix
```

Comment (ivanholmes) [reply haiku-apps/pdftranslator/pdftranslator-1.0.4.recipe]: No worries. Still working on DocumentViewer. It builds now but gets stuck waiting for a mutex... will investigate further later on.

**Comment by korli** [comment]
depends on #6823

**Comment by Begasus** [comment]
Build failure on buildmaster, I guess a change from R1B3 vs nightly? (builds fine on nightly here):

https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/2658/builds/77954.log
https://build.haiku-os.org/buildmaster/master/x86_64/logviewer.html?buildruns/6984/builds/77225.log

**Comment by augiedoggie** [comment]
The easiest fix is to force `g++` during the compile.  This will automatically link libstdc++ and fix the errors.  I have built this on beta3 by changing the line to `make CC=g++ $jobArgs`

**Pull request merged**